howto/android/access-instance.md

```diff
@@ -0,0 +1,156 @@
+(howto-access-instance)=
+# How to access an instance
+
+For certain scenarios, accessing an individual instance for debugging or development might be necessary. You can do this by:
+
+- Run `anbox-connect` for secure ADB connections.
+- Expose ADB service upon Anbox instance creation.
+
+## Access the Android instance using anbox-connect
+
+Since the 1.23.0 release, Anbox Cloud allows users to create a secure ADB connection through the `anbox-connect` command without exposing the [ADB service](https://anbox-cloud.io/docs/howto/instance/expose-services). This method is highly recommended for securely connecting to remote Android instances, offering a safe and efficient means of managing ADB connections.
+
+### Install `anbox-connect` snap
+
+First, install the `anbox-connect` snap from the snap store:
+
+    snap install anbox-connect
+
+### Generate a presigned connection URL
+
+Generate a presigned connection URL in the following ways:
+
+- **From the Dashboard**:
+  Navigate to the `Instances` view from the left panel, locate a running instance, click the ![Connect ADB|16x16](https://assets.ubuntu.com/v1/51973cea-adb-connect-icon.png) icon button, then click `Authorize` in the popup dialog and copy the command provided.
+
+- **From the Command Line**:
+  * For a full Anbox Cloud deployment:
+
+        anbox-stream-gateway session share <session_id>
+
+  * For the Anbox Cloud Appliance:
+
+        anbox-cloud-appliance.gateway session share <session_id>
+
+  This command will return a presigned URL that you can use to connect to the remote Android instance.
+
+```{note}
+Each presigned URL can only be used to establish a single ADB connection. If multiple users attempt to use the same presigned URL, any existing ADB connection will be interrupted to allow the new request to succeed.
+```
+
+### Connect to the Instance
+
+Open a terminal and use the `anbox-connect` command to establish a secure ADB connection to the remote Android instance:
+
+    anbox-connect <connection_url>
+
+```{note}
+If the Anbox Stream Gateway or Anbox Cloud Appliance uses self-signed certificates, include the `--skip-tls-verify` or `-k` parameter to skip TLS certificate verification.
+```
+
+After the ADB channel is established, the following output will be displayed:
+
+    Access to the remote Anbox Cloud instance over ADB is now possible. Please run
+
+        $ adb connect 127.0.0.1:32985
+
+Lastly, follow the prompt in the command line output and run:
+
+    adb connect 127.0.0.1:32985
+
+to setup the ADB connection.
+
+```{note}
+The `anbox-connect` command sets up a secure ADB channel and routes traffic between your local machine and the remote Anbox instance. Therefore, it must be kept running to maintain the ADB connection. Do not abort the command once the connection is established.
+```
+
+(sec-expose-adb-service)=
+## Expose ADB service upon Anbox instance creation
```

Comment:
Two suggestions:
1. It would be a good idea to call out that we use a third party tool, scrcpy: 
Something similar to what was existing - "The AMS services enable connecting remotely over a network to instance via [scrcpy](https://github.com/Genymobile/scrcpy).Scrcpy provides a simple way to display and control instances running on Anbox Cloud remotely."

I understand users will realise when they are performing the task but it is better to let them know before they start the task.
2. Although this method is not preferred as it not secure, I think we should mention that this method provides graphical access

Reply:
@adglkh I have added suggestions to implement what I meant in this comment. Please have a look, everything else is good.